hyperspy/io_plugins/fei.py

```diff
@@ -316,6 +316,97 @@ def get_xml_info_from_emi(emi_file):
     return objects[:-1]
 
 
+def get_calibration_from_position(position):
+    offset = position[0]
+    for i, x in enumerate(position):
+        if x != position[0]:
+            break
+    if i == len(position) - 1:
```

Comment:
Will this be correct if it breaks on the last iteration? Consider using the `for ... else` syntax, or commenting code. Maybe add a docstring to the function?